sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -49,12 +58,21 @@ export interface GetTokenOptions {
     };
 }
 
+// @public
+export type HttpMethods = "GET" | "PUT" | "POST" | "DELETE" | "PATCH" | "HEAD" | "OPTIONS" | "TRACE";
```

Comment:
Does this need to be public?

Reply:
Yes this will need to be public, since we didn't want to introduce a circular dependency on core packages. We need to redefine it here. But this type needs to be public to define the type for the field `resourceRequestMethod` in popTokenOptions

Reply:
Not sure if necessary for this PR, but perhaps it would be useful to declare this type in core-util and re-export from the places where we are using it currently to avoid the duplication?

Reply:
Done